data/test_e2ebot/tests/test_stories.yml

```
@@ -0,0 +1,12 @@
+stories:
```

to avoid warnings in the CI
```suggestion
version: "2.0"

stories:
```